documentation/design/output.md

```diff
@@ -8,6 +8,8 @@ Github audit plugin will be providing output in following json format:
 {
     "type": "commit",
     "repo_type":"github",
+    "repo_name":"test_repo",
+    "repo_url":"https://github.com/testurl",
     "created_at": "2022-08-30T16:25:04Z",
```

Comment:
Better to include time as timestamp. It is upto the user to display in their required format.